waveform_collection/server.py

```diff
@@ -74,26 +74,31 @@ def gather_waveforms(source, network, station, location, channel, starttime,
             ``http://10.30.5.10:8080`` if using VPN)
         watc_username (str): Username for WATC FDSN server
         watc_password (str): Password for WATC FDSN server
+        verbose (bool): If `False`, all print statements will be blocked. 
+            Default is `True`.
 
     Returns:
         :class:`~obspy.core.stream.Stream` containing gathered waveforms. If
         `return_failed_stations` is `True`, additionally returns a list
         containing station codes that were requested but not downloaded
     """
-
+    
+    # Set up log function if a verbose output is desired
+    log = print if verbose else lambda value, *args, **kw: None
```

Comment:
I think that `*args` includes `value` — can you try if this works? I think this is the canonical "empty" lambda function form. Also note that I'm using `**kwargs` here as this is the standard way to write this.
```suggestion
    log = print if verbose else lambda *args, **kwargs: None
```